include/usb/usb_common.h

```diff
@@ -60,12 +60,12 @@
 
 /* Descriptor type */
 #define USB_DEVICE_DESC			0x01
```

Comment:
Seems weird that only selected descriptor types get the 'U' specifier?

Reply:
Hi David, Yes the others were not flagged yet and that is because it will be flagged only when these constants are used in the context where the two operands are not of the same type. But for consistency, I have marked all the descriptor types with "Y" specifier now. Please review again and let me know  if it looks fine now. Thanks